src/reducer.test.js

```diff
@@ -125,4 +134,47 @@ describe('reducer', () => {
       });
     });
   });
+
+  describe('changeLoginField', () => {
+    it('changes login field', () => {
+      const initialState = {
+        loginFields: {},
+      };
+
+      const state = reducer(
+        initialState,
+        changeLoginField({ name: 'email', value: 'test' }),
+      );
+
+      expect(state.loginFields).toEqual({
+        email: 'test',
+      });
+    });
+  });
+
+  describe('setAccessToken', () => {
+    const initialState = {
+      accessToken: '',
+    };
+
+    const state = reducer(initialState, setAccessToken('TOKEN'));
+
+    expect(state.accessToken).toBe('TOKEN');
```

Comment:
```suggestion
    const { accessToken } = reducer(initialState, setAccessToken('TOKEN'));

    expect(accessToken).toBe('TOKEN');
```

구조 분해 할당을 적용해줘도 좋을 것 같아요~ 다른 부분들도 구조분해할당이 가능하다면 수정해주세요 😄